src/nl/rubensten/texifyidea/util/Labels.kt

```diff
@@ -76,13 +93,28 @@ fun PsiFile.findLabels(): Collection<PsiElement> {
  * @return The found label commands.
  */
 fun Project.findLabels(): Collection<PsiElement> {
+    val commandNames = TexifySettings.getInstance().labelCommands
+
     val commands = LatexCommandsIndex.getItems(this)
+    val commandsFiltered = commands.filter { commandNames.containsKey(it.name) }
     val bibtexIds = BibtexIdIndex.getIndexedIds(this)
-    val result = ArrayList<PsiElement>(commands)
+    val result = ArrayList<PsiElement>(commandsFiltered)
     result.addAll(bibtexIds)
     return result.findLabels()
 }
 
+/**
+ * Finds all specified bibtex entries
+ */
+fun PsiFile.findBibtexItems(): Collection<PsiElement> {
+    val bibtex = BibtexIdIndex.getIndexedIdsInFileSet(this)
+    val bibitem = findBibitemCommands().toList()
+    return (bibtex + bibitem)
+}
+
+fun PsiFile.findBibitemCommands(): Sequence<LatexCommands> = LatexCommandsIndex.getItemsInFileSet(this).asSequence()
```

Comment:
No documentation.

Reply:
fixed in commit e43885c396a6aec2c17f9d63fb39fa2ae1c7c550